chag

```diff
@@ -173,7 +214,7 @@ latest() {
 
 # Get the line number of a specific changelog entry.
 find_entry() {
-  result=$(entries_and_lines | grep -E ":${1//./\\.}$" | cut -d ':' -f 1)
+  result=$(entries_and_lines | grep "${1//./\\.}" | cut -d ':' -f 1)
```

Comment:
I'm repeating my opinion: it works and in case that it turns out that it doesn't in some circumstances we can fix it when it comes to that.